File: src/test/java/jaba/lists/linkedlists/singlelinkedlists/JosephusProblemTest.java
package jaba.lists.linkedlists.singlelinkedlists;

import jaba.lists.linkedlists.singlelinkedlists.recursive.Node;
import org.junit.Assert;
import org.junit.Test;

/**
 * Created by Administrador on 22/05/2017.
 */
public class JosephusProblemTest {
    @Test
    public void getSurvivor() {
        final JosephusProblem josephusProblem = new JosephusProblem(9,5);
        final Node survivor = josephusProblem.getSurvivor();
        System.out.println(survivor);
        Assert.assertNotNull(survivor);
    }

}

File: src/test/java/katlin/linkedlists/singlelinkedlists/JosephusProblemTest.kt
package katlin.linkedlists.singlelinkedlists

import junit.framework.Assert.assertNotNull
import org.junit.Test

/**
 * Created by Administrador on 24/05/2017.
 */
class JosephusProblemTest {
    @Test
    fun getSurvivor() {
        val josephusProblem = JosephusProblem(10, 3)
        val survivor = josephusProblem.survivor
        println(survivor)
        assertNotNull(survivor)
    }

}